mettle/src/main.c

```diff
@@ -102,17 +102,14 @@ static int parse_cmdline(int argc, char * const argv[], struct mettle *m, int fl
 		case 'd':
 			if (optarg) {
```

Comment:
Do we still need this guard now that option argument is required (ditto for `-b`)?

Reply:
Nope, it never gets hit because getopt_long bails early. I was lazy because there were some scoped variables in the block, but I'll remove the if